packages/react-components/src/templates/NetworkPageHeader.tsx

```diff
@@ -81,19 +66,28 @@ const NetworkPageHeader: React.FC<NetworkPageHeaderProps> = ({
           onChange={onChangeToggle}
           position={page === 'teams' ? 'left' : 'right'}
         />
-        <div css={searchContainerStyles}>
-          <SearchField
-            value={query}
-            placeholder={
-              page === 'users' ? 'Search for someone…' : 'Search for a team…'
-            }
-            onChange={onChangeSearch}
-          />
-          <Button enabled={false}>
-            {filterIcon}
-            <span css={buttonTextStyles}>Filters</span>
-          </Button>
-        </div>
+        <SearchControls
+          onChangeSearch={onChangeSearch}
+          placeholder={
+            page === 'users' ? 'Search for someone…' : 'Search for a team…'
+          }
+          query={query}
+          filterEnabled={page === 'users'}
+          filterOptions={
+            page === 'users'
+              ? [
+                  { label: 'Lead PI', value: '' },
```

Comment:
@fampinheiro just tagging so you are aware since this looks like it's gonna be an interesting architecture challenge :D

Reply:
I could pass this in from Frontend?

Reply:
Not sure that improves anything - it's okay to rely on the model here, it's just that we have a hardcoded role list in many places